pkg/k8s/apis/cilium.io/v1alpha1/register_podinfo.go

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
let's use cilium.io/v1alpha1/register.go instead of adding a new file

Reply:
done.